Supported allele check and show (#70)
nf-core/metapep: docs/usage.md

The pipeline needs to handle large amounts of data, depending on the size and number of microbiomes the user has defined in the input. To handle these data the pipeline mainly uses python scripts and the python module pandas. As the data needs to be compared memory consumption is currently peaking at around 150 GB for the full-size test, but can easily be higher depending on the input. If the memory is still an issue one can try to reduce the chunk sizes for high memory consuming processes. The parameters are: `--proc_chunk_size` for the preprocessing of the peptides prior to the epitope prediction in `SPLIT_PRED_TASKS`, `--pred_chunk_size` for the epitope prediction process `PREDICT_EPITOPES` and `--ds_prep_chunk_size` for the downstream analysis processes `PREPARE_ENTITY_BINDING_RATIOS` and `PREPARE_SCORE_DISTRIBUTION`.

### Supported allele models

> The pipeline predicts epitopes based on a specific peptide of lenth x on a specific allele of MHC class I or class II. As the prediction is performed by external tools, the user is restricted to the corresponding combinations the external tools are offering. Therefore, the metapep pipeline comes with a functionality to output all supported alleles and supported lengths of the supported external tools, which is invoked by:

Reviewer: ```suggestion
The pipeline predicts epitopes for specific peptide lengths and for specific alleles of MHC class I or class II. As the prediction is performed by external tools, the user is restricted to the corresponding combinations the external tools are offering. Therefore, the metapep pipeline comes with a functionality to output all supported alleles and supported lengths of the supported external tools, which is invoked by:
```
Maybe?

> `nextflow run nf-core/metapep -profile docker --outdir metapep_alleles --supported_allele_information`
> 
> The outdir can in this case be changed to another directory if needs be. More on the output can be found at https://nf-co.re/metapep/dev/output#supported-allele-models

Reviewer: ```suggestion
`nextflow run nf-core/metapep -profile docker --outdir <OUTDIR> --supported_allele_information`

More on the output can be found at https://nf-co.re/metapep/dev/output#supported-allele-models
```

Or `--outdir <MODEL_OUTDIR>`

Author: I think I would go with OUTDIR to make it consistent with the main pipeline and to stress this is the same directory.
